cmd/support-diag.go

```diff
@@ -325,9 +326,23 @@ func fetchServerDiagInfo(ctx *cli.Context, client *madmin.AdminClient) (interfac
 			admin(len(info.Minio.Info.Servers) > 0)
 	}
 
+	progressV3 := func(info madmin.HealthInfoV3) {
```

Comment:
No difference in entries for `progressV2` and `progressV3`? Shouldn't we add `netInfo` here?

Reply:
V3 is the existing version. Previous `progress` (that was handling v3) becomes `progressV3` and new `progress` (that handles v4) now has `netInfo` also.